### DISCUSSION TITLE
Comments are only possible for people who have edit-access

### DISCUSSION THREAD

**Discussion by BK-Countly** [outline/outline#5125] [General]
It would be way helpful if everyone had the chance to comment on anything or if there were edit-access-to-comments levels as we have other teams in need of commenting on docs they are not the creators of.

**Comment by tommoor** [comment]
Thanks, this one is a known limitation but we should track it. For now, viewers can leave document-level comments in the sidebar but not comment on selections of text.

**Comment by Darkdragon84** [comment]
this would be a very useful feature!
